### PR TITLE
feat(email): Add primary color

### PR DESCRIPTION
**Pull request by aXenDeveloper**

### CODE REVIEW

File: backend/src/functions/colors.ts [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+export const hexRegex = /^#?([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/;
+export const hslRegex = /^hsl\(\s*\d{1,3}\s*,\s*\d{1,3}%\s*,\s*\d{1,3}%\s*\)$/;
+export const rgbWithCommaRegex =
+  /^rgb\(\s*\d{1,3}\s*,\s*\d{1,3}\s*,\s*\d{1,3}\s*\)$/;
+export const rgbWithoutCommaRegex =
+  /^rgb\(\s*\d{1,3}\s+\d{1,3}\s+\d{1,3}\s*\)$/;
+
+export const checkColorType = (strColor: string): CheckColorReturn | null => {
+  if (hexRegex.test(strColor)) {
```
Comment (AndreSoftwareDeveloper) [inline backend/src/functions/colors.ts:155]: redundant braces

File: backend/src/plugins/core/admin/email/settings/show/show.service.ts [hunk 2]
```diff
@@ -5,22 +5,29 @@ import { Injectable } from "@nestjs/common";
...
+    const configSettings = getConfigFile();
+
     if (!fs.existsSync(this.path)) {
       return {
         smtp_host: "",
         smtp_user: "",
         smtp_secure: false,
-        smtp_port: 587
+        smtp_port: 587,
+        ...configSettings.settings.email
```
Comment (AndreSoftwareDeveloper) [inline backend/src/plugins/core/admin/email/settings/show/show.service.ts]: should be in separate variable, especially where used again in return statement

**Review by AndreSoftwareDeveloper — COMMENTED** [review]
refactor please

File: backend/src/functions/colors.ts (hunk 1 above)
Comment (aXenDeveloper) [reply backend/src/functions/colors.ts:155]: In regex?

File: backend/src/plugins/core/admin/email/settings/show/show.service.ts (hunk 2 above)
Comment (aXenDeveloper) [reply backend/src/plugins/core/admin/email/settings/show/show.service.ts]: Show shows everything from email settings so it's fine here.

**Review by AndreSoftwareDeveloper — APPROVED** [review]

**Pull request merged**